docs/uppmax.md

```diff
@@ -74,14 +79,197 @@ It is not suitable for use with real data.
 
 To use it, submit with `-profile uppmax,devel`.
 
-## Running on Bianca
+## Running on bianca
 
-For security reasons, there is no internet access on Bianca so you can't download from or upload files to the cluster directly. Before running a nf-core pipeline on Bianca you will first have to download the pipeline and singularity images needed elsewhere and transfer them via the wharf area to your Bianca project.
+> :warning: For more information, please follow the following guides:
+>
+> * [UPPMAX `bianca` user guide](http://uppmax.uu.se/support/user-guides/bianca-user-guide/).
+> * [nf-core guide for running offline](https://nf-co.re/usage/offline)
+> * [nf-core `tools` guide for downloading pipelines for offline use](https://nf-co.re/tools#downloading-pipelines-for-offline-use).
+> * [UPPMAX `Singularity` guide](https://www.uppmax.uu.se/support-sv/user-guides/singularity-user-guide/).
 
-You can follow the guide for downloading pipelines [for offline use](https://nf-co.re/tools#downloading-pipelines-for-offline-use). Note that you will have to download the singularity images as well.
+For security reasons, there is no internet access on `bianca` so you can't download from or upload files to the cluster directly.
+Before running a nf-core pipeline on `bianca` you will first have to download the pipeline and singularity images needed elsewhere and transfer them via the `wharf` area to your own `bianca` project.
+
+In this guide, we use `rackham` to download and transfer files to the `wharf` area, but it can also be done on your own computer.
+If you use `rackham` to download the pipeline and the singularity containers, we recommend using an interactive session (cf [interactive guide](https://www.uppmax.uu.se/support/faq/running-jobs-faq/how-can-i-run-interactively-on-a-compute-node/)), which is what we do in the following guide.
+
+### Download and install Nextflow
+
+You can use the `Nextflow` UPPMAX provided `module`, but if necessary, you can also download a more recent version.
+
+```bash
+# Connect to bianca
+$ ssh -A <USER>-<BIANCA_PROJECT>@bianca.uppmax.uu.se
+
+# See the available versions for the module
+module spider Nextflow
```

Comment:
I haven't verified but don't think the spider will see Nextflow without having bioinfo-tools loaded.